src/inmanta/config.py

```diff
@@ -166,6 +170,18 @@ def _reset(cls) -> None:
         cls.__instance = None
         cls._config_dir = None
         cls._min_c_config_file = None
+        cls._config_updated()
+
+    @classmethod
+    def _config_updated(cls) -> None:
```

Comment:
I think I'd prefer a listener-type interface, also to prevent the import issue. But we can do that if a second use case pops up.